src/test/java/org/assertj/core/error/NoElementsShouldSatisfy_create_Test.java

```diff
@@ -0,0 +1,23 @@
+package org.assertj.core.error;
+
+import org.assertj.core.description.TextDescription;
+import org.assertj.core.presentation.StandardRepresentation;
+import org.junit.Test;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.error.NoElementsShouldSatisfy.noElementsShouldSatisfy;
+import static org.assertj.core.util.Lists.newArrayList;
+
+public class NoElementsShouldSatisfy_create_Test {
+
+    @Test
+    public void should_create_error_message_any() {
```

Comment:
`should_create_error_message_any` ? please double check the code you are writing